### PR TITLE
Arista/show module

### PR DESCRIPTION
**Pull request by Sandeepsr**
##### ISSUE TYPE

 - New Template Pull Request

##### COMPONENT
adding arista_eos_show_module.template

##### SUMMARY
adding arista_eos_show_module.template

```

```

### CODE REVIEW

File: templates/arista_eos_show_module.template [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Value MODULE (\S+)
+Value Required PORTS (\d+)
+Value CARD (((\S+-port\s+\S+)|(\S+))?)
+Value TYPE (\S+)
+Value MODEL (\S+)
+Value SERIAL_NUM (\S+)
+Value Fillup MAC_ADD (\S+\s+-\s+\S+)
```
[the remark this answers is not in the document]
Comment (Sandeepsr) [reply templates/arista_eos_show_module.template]: @itdependsnetworks , I will test suggestion and let you know

[the remark this answers is not in the document]
Comment (itdependsnetworks) [reply templates/arista_eos_show_module.template]: Let me enlist others, @jmcgill298 and @jedelman8 any thoughts? to me a single data field should represent a single piece of information.

[the remark this answers is not in the document]
Comment (jmcgill298) [reply templates/arista_eos_show_module.template]: I would rather see `MAC_START` and `MAC_END`. It is easier to recompile data that is parsed with more granularity than to parse data that contains multiple pieces of information.

**Comment by jmcgill298** [comment]
#343

**Pull request closed without merging**